pkg/kubelet/volumemanager/BUILD

```diff
@@ -1,93 +0,0 @@
-package(default_visibility = ["//visibility:public"])
```

Comment:
All BUILD files should be already removed, please rebase to master branch.

Reply:
Thanks for reminding, I have fixed it. Thanks.